src/__tests__/transformer.test.js

```diff
@@ -1,27 +1,100 @@
-import fs from 'fs'
-import path from 'path'
-import transformer from '../transformer'
-import sharedTests from './fixtures/shared/commonTransformerTests'
-
-const runTransformer = (filename, options) => {
-  const process = transformer.createTransformer(options).process
-  const path = require.resolve(`./fixtures/${filename}.svelte`)
-  const source = fs.readFileSync(path).toString()
-  const result = process(source, path)
+/**
+ * @jest-environment jsdom
+ */
+import { readFileSync } from 'fs'
+import { dirname } from 'path'
+import { fileURLToPath } from 'url'
+import { jest } from '@jest/globals'
+
+import createTransformer from '../transformer'
+
+// Node API __dirname is missing in Jest
```

Comment:
it's not Jest, but rather ESM
```suggestion
// Node API __dirname is missing in ESM
```